Casbin/Adapter/File/FileAdapter.cs

```diff
@@ -34,15 +34,21 @@ public FileAdapter(Stream inputStream)
             }
         }
 
-        public void LoadPolicy(IPolicyStore model)
+        public FileAdapter(Stream inputStream, Stream outputStream)
         {
-            if (string.IsNullOrWhiteSpace(FilePath) is false)
+            try
+            {
+                _byteArrayInputStream = new StreamReader(inputStream);
+                _byteArrayOutputStream = new StreamWriter(outputStream);
```

Comment:
Here maybe an legacy problem.

 We need to avoid storing StreamReader or StreamWriter in a field, there is a lot of risks to call `Dispose` in time.

Reply:
ok, I'll change it later

Reply:
I recommend creating a new Adapter named `StreamAdapter`.